.changeset/nine-rats-kneel.md

```diff
@@ -0,0 +1,5 @@
+---
+'@spreadshirt/backstage-plugin-s3-viewer-backend': minor
+---
+
+Add AuditService to replace default logging on Download Events, also provided optional configuration settings.
```

Comment:
Just one note here. The `minor` is the best approach here for sure. However, I would also add a note mentioning that this change is a potential breaking change to those people calling the `S3Builder` constructor.